fact/coordinates/representation.py

```diff
@@ -0,0 +1,78 @@
+from astropy.coordinates import BaseRepresentation, CartesianRepresentation
+import astropy.units as u
+from collections import OrderedDict
+from astropy.utils.compat.numpy import broadcast_arrays
+
+
+class PlanarRepresentation(BaseRepresentation):
```

Comment:
Coordinates need to have a representation, astropy does not provide a 2d Cartesian representation, only 3d. This is basically copy/paste from ctapipe (with a fixed bug).